packages/govern-core/contracts/test/TestToken.sol

```diff
@@ -0,0 +1,179 @@
+/*
```

Comment:
Use token contract from ``govern-token`` and add this package as ``devDependency``